libraries/AC_CustomControl/AC_CustomControl.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+/// @file    AC_CustomControl.h
+/// @brief   ArduCopter custom control library
+
+#include <AP_Common/AP_Common.h>
+#include <AP_Param/AP_Param.h>
+#include <AP_Vehicle/AP_Vehicle.h>
+#include <AP_AHRS/AP_AHRS_View.h>
+#include <AC_AttitudeControl/AC_AttitudeControl_Multi.h>
+#include <AP_Motors/AP_Motors.h>
+#include <AP_Logger/AP_Logger.h>
+
+#ifndef CUSTOMCONTROL_ENABLED
+#define CUSTOMCONTROL_ENABLED (!HAL_MINIMIZE_FEATURES && !defined(HAL_BUILD_AP_PERIPH) && BOARD_FLASH_SIZE > 1024 && 0)
```

Comment:
```suggestion
#define CUSTOMCONTROL_ENABLED (CONFIG_HAL_BOARD == HAL_BOARD_SITL)
```
Maybe enable for SITL, should so at least the code is built.

Reply:
How about this? We only allow compilation on a 2 MB flash board or SITL. 
`#define CUSTOMCONTROL_ENABLED (!HAL_MINIMIZE_FEATURES && !defined(HAL_BUILD_AP_PERIPH) && BOARD_FLASH_SIZE > 1024 || CONFIG_HAL_BOARD == HAL_BOARD_SITL)
`